teams/views.py

```diff
@@ -13,6 +13,7 @@
 from accounts.decorators import can_access
 from accounts.models import UserType
 from accounts.lists import TIMEZONE_CHOICES
+from competencies.models import Competency
 from hackathon.models import Hackathon, HackTeam, HackProject
 from teams.helpers import choose_team_sizes, group_participants,\
```

Comment:
perhaps:

```python
from teams.helpers import (
    choose_team_sizes, group_participants,
    choose_team_levels, find_all_combinations,
    distribute_participants_to_teams,
    create_teams_in_view, update_team_participants,
    calculate_timezone_offset)
```

Reply:
whoops just realized this isn't one of your changes.. but would be a nice cleanup

Reply:
Changed as suggested.